traffic_portal/app/src/common/api/CoordinateService.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var CoordinateService = function($http, $q, Restangular, locationUtils, messageModel, ENV) {
+
+    this.getCoordinates = function(queryParams) {
+        return Restangular.all('coordinates').getList(queryParams);
+    };
+
+	this.createCoordinate = function(coordinate) {
+		var request = $q.defer();
+
+		$http.post(ENV.api['root'] + "coordinates", coordinate)
+			.then(
+				function(response) {
+                    messageModel.setMessages(response.data.alerts, true);
```

Comment:
nitpick - formatting

Reply:
Shoot, I had to switch to a new IDE recently, and it appears to mix tabs and spaces in the same file. I'll keep an eye on formatting next time.